providers/os/resources/fstab.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+package resources
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	"strconv"
+	"strings"
+
+	"go.mondoo.com/cnquery/v11/llx"
+	"go.mondoo.com/cnquery/v11/providers/os/connection/shared"
+)
+
+func (f *mqlFstab) entries() ([]any, error) {
+	conn := f.MqlRuntime.Connection.(shared.Connection)
```

Comment:
we should check if its castable

Reply:
Do you mean generally or just in this particular case? I "copied" this over from iptables

Reply:
best case: generally, but just fixing this case should do it 😄